source/app/views/results.erb

```diff
@@ -0,0 +1,21 @@
+<h1>Results</h1>
+
+<p>Winner is <%= @winner %></p>
+<p>With: <%= @winning_token %> </p>
+<p>Loser is <%= @loser %></p>
+<p>With: <%= @losing_token %> </p>
+<br><br>
+
+<%if @winner=="Player One" %>
+  <p><%= @winner %> has won <%= @winner_wins %> times.</p>
+  <p><%= @winner %> has won with <%= @winning_token %> <%= Game.where(winner_id: @game.winner_id, choice_one_id: @game.choice_one_id).count %> times.</p><br>
```

Comment:
This is great. I like that you're comfortable using `where` and chaining ActiveRecord methods.

I'd recommend moving the logic to the model. Let's chat more on Monday!